cmd/juju/cloud/add.go

```diff
@@ -137,6 +148,14 @@ func (c *addCloudCommand) runInteractive(ctxt *cmd.Context) error {
 		return errors.Trace(err)
 	}
 
+	pollster.VerifyURLs = func(s string) (ok bool, msg string, err error) {
+		err = c.Ping(provider, s)
```

Comment:
I don't think so, I think we can have a known limitation and come back and get it worked. Worst case we need to patch to have it always return success for now and we add the url checking validation as a follow up step. Actually, let's go that route for this PR. Have it always succeed and then come back at it as a known issue in follow up.